charts/shoot-core/components/charts/monitoring/charts/blackbox-exporter/templates/pdb.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: blackbox-exporter-pdb
+spec:
+  minAvailable: 1
```

Comment:
Yeah, we should rather invert it and use `maxUnavailable: 1`

Reply:
Would `maxUnavailable: 1` help when we have only 1 node but still 2 replicas of the blackbox-exporter? I'm not sure: should the number of replicas be adapted in that case?

Reply:
Let's say we want to adjust the number of replicas depending if it's a single node cluster or not, should we use a values.yaml parameter in the umbrella chart shoot-core/components ?

Reply:
Hi @bd3lage, I'm not sure how to best avoid the edge case when the pod disruption budget prevents draining the last/only node where both replicas are running.

I was also considering leaving out the pdb and just benefiting from the improvement of having 2 replicas, so that the disruptions due to voluntary evacuation will be less likely / shorter, even if not avoided altogether.

@timebertt, what do you think?

Reply:
My idea was, I configured the BBX PDB same as the CoreDNS, and the CoreDNS PDB does not prevent us to properly drain the last/only node.

Reply:
Hi Rafael, thank you for the example with coredns. :)
I think there should be no difference with respect to the PDB, so if the pdb of coredns haven't caused troubles when draining the last node in the past months, then my hypothetical concern should not be an issue and the PR looks good to me.